__tests__/util/request-manager.js

```diff
@@ -34,3 +34,38 @@ test('RequestManager.request with cafile', async () => {
   }
   expect(body).toBe('ok');
 });
+
+test('RequestManager.execute Request 403 error', async () => {
+  const config = await createConfig({});
+  jest.mock('request', (factory) => (options) => {
+    options.callback('', {statusCode: 403}, '');
+    return {
+      on: () => {},
+    };
+  });
+  await config.requestManager.execute({
+    params: {url: `https://localhost:port/?nocache`, headers: {Connection: 'close'}},
+    resolve: (body) => {},
+    reject: (err) => {
+      expect(err.message).toBe('https://localhost:port/?nocache: Request "https://localhost:port/?nocache" returned a 403');
```

Comment:
I was wondering if I can make jest to wait for the execution of this expect (like the done from mocha) to ensure the execution of it?

Also here the url is set it twice because the reject is going to set the message like `err.message = `${params.url}: ${err.message}`;` so I dont know if you guys think the url should be removed from the original message